guides/common/modules/proc_overriding-ansible-variables-in-project.adoc

```diff
@@ -6,6 +6,11 @@ If you run Ansible roles in {Project}, you can use {Project} to override Ansible
 The following procedure refers to hosts and host groups.
 For more information, see {ManagingHostsDocURL}[_{ManagingHostsDocTitle}_].
 
+[WARNING]
+====
+To use overridden Ansible variables, a user must have a role that allows them to see those variables.
```

Comment:
Can you please take a look at line 53? https://github.com/theforeman/foreman-documentation/blob/6243070f7c89f52704201590ebf008fc7ad4fa79/guides/common/modules/proc_overriding-ansible-variables-in-project.adoc?plain=1#L53

Both your addition, and line 53, talk about how to use the overridden variables, right? I think you should document both together.

Reply:
I am not sure about that. That looks like next step after the procedure, while this warning is more of a prerequisite. Which reminds me, I should probably just put it into prerequisites instead of a warning. Would that be better?

Reply:
@asteflova

Reply:
Oh, thanks for explaining! Yes, adding it to `.Prerequisites` is a great idea then.

Reply:
Alright, it is now in prerequisites.